substrate/client/chain-spec/src/genesis_config_builder.rs

```diff
@@ -81,7 +81,7 @@ where
 			.0
 	}
 
-	/// Returns the default `GenesisConfig` provided by the `runtime`.
+	/// Returns the default `RuntimeGenesisConfig` provided by the `runtime`.
```

Comment:
Returns a json representation of the default `RuntimeGenesisConfig` provided by the `runtime`.